modules/backend/lang/de/lang.php

```diff
@@ -516,44 +547,46 @@
         'required_match_column_error' => 'Bitte stellen Sie eine Verbindung zwischen den benötigten Feldern :label her.',
-        'empty_export_columns_error' => 'Bitte geben Sie einige Reihen zum exportieren an.',
-        'behavior_missing_uselist_error' => 'Sie müssen das Controller-Verhalten "ListController" implementieren und "useList" Option aktivieren.',
-        'missing_model_class_error' => 'Bitte geben Sie die spezifische "modelClass" Eigenschaft für :type an.',
-        'missing_column_id_error' => 'Fehleneder Reihen Identifier',
-        'unknown_column_error' => 'Unbekannte Reihe',
-        'encoding_not_supported_error' => 'Qulldatei Encoding wurde nicht bekannt oder ist nicht bekannt. Bitte wählen Sie ein Dateiformat mit ein entsprechendes Encoding um es zu importieren.',
-        'encoding_format' => 'Datei encoding',
+        'empty_export_columns_error' => 'Bitte geben Sie einige Spalten für den Export an.',
+        'behavior_missing_uselist_error' => 'Sie müssen das Controller-Verhalten "ListController" implementieren und die "useList"-Option aktivieren.',
+        'missing_model_class_error' => 'Bitte definieren Sie Eigenschaft in der Model-Klasse für :type an.',
+        'missing_column_id_error' => 'Fehlenede Spaltenkennung',
+        'unknown_column_error' => 'Unbekannte Spalte',
+        'encoding_not_supported_error' => 'Das Encoding der Quelldatei wurde nicht erkannt. Bitte wählen Sie ein Dateiformat mit einem entsprechenden Encoding, um eine Datei zu importieren.',
+        'encoding_format' => 'Datei-Encoding',
         'encodings' => [
             'utf_8' => 'UTF-8',
             'us_ascii' => 'US-ASCII',
-            'iso_8859_1' => 'ISO-8859-1 (Latin-1, Western European)',
-            'iso_8859_2' => 'ISO-8859-2 (Latin-2, Central European)',
-            'iso_8859_3' => 'ISO-8859-3 (Latin-3, South European)',
-            'iso_8859_4' => 'ISO-8859-4 (Latin-4, North European)',
-            'iso_8859_5' => 'ISO-8859-5 (Latin, Cyrillic)',
-            'iso_8859_6' => 'ISO-8859-6 (Latin, Arabic)',
-            'iso_8859_7' => 'ISO-8859-7 (Latin, Greek)',
-            'iso_8859_8' => 'ISO-8859-8 (Latin, Hebrew)',
-            'iso_8859_9' => 'ISO-8859-9 (Latin-5, Turkish)',
-            'iso_8859_10' => 'ISO-8859-10 (Latin-6, Nordic)',
+            'iso_8859_1' => 'ISO-8859-1 (Latin-1, Westeuropäisch)',
+            'iso_8859_2' => 'ISO-8859-2 (Latin-2, Mitteleuropäisch)',
+            'iso_8859_3' => 'ISO-8859-3 (Latin-3, Südeuropäisch)',
+            'iso_8859_4' => 'ISO-8859-4 (Latin-4, Nordeuropäisch)',
+            'iso_8859_5' => 'ISO-8859-5 (Latin, Kyrillisch)',
+            'iso_8859_6' => 'ISO-8859-6 (Latin, Arabisch)',
+            'iso_8859_7' => 'ISO-8859-7 (Latin, Griechisch)',
+            'iso_8859_8' => 'ISO-8859-8 (Latin, Hebräisch)',
+            'iso_8859_9' => 'ISO-8859-9 (Latin-5, Türkisch)',
+            'iso_8859_10' => 'ISO-8859-10 (Latin-6, Nordisch)',
```

Comment:
Are these typically translated in other systems? Our general approach is to not translate them here.

Reply:
@LukeTowers it's hard to find a CMS or e-commerce software with encoding settings.

I took a look to wikipedia and common windows tools. Indeed in german language it seems to be common to translate the encoding partial standard. You can see in the collected screenshots below.

![translated](https://user-images.githubusercontent.com/25099134/182345129-2fd2d0b0-d67a-4c0a-a059-fe734c399810.jpg)

Should I change it back anyway to English? I guess to leave it in English should not be a problem for users. I guess that a user who knows what encoding is, should even know this few words in English. Tell me how you like it.

Reply:
In that case I'm fine with leaving these translated to German, the precedent seems fine to me.